fastlane/metadata/id/name.txt

```diff
@@ -1 +0,0 @@
-Simplenote - Catatan dan Tugas
```

Comment:
Be sure to manually revert the copy in ASC for that `id` locale – as removing the file here means that `fastlane deliver` will just not touch/update the app name field for that locale in ASC anymore, but it was likely already updated in ASC during last release.

(I might be wrong here, i.e. maybe instead `fastlane deliver` clears the name for those locales in ASC to make them falls back to the English copy for locales which don't have a `name.txt` file, as opposed to leave them untouched? But better double-check anyway)